components/Datasets/DescribeDataset.vue

```diff
@@ -288,6 +294,55 @@ Elle aide les utilisateurs à comprendre rapidement ce qu’il contient et amél
             >
               {{ getFirstWarning("description_short") }}
             </SimpleBanner>
+            <div class="flex items-center gap-4 mt-2 mb-3">
+              <Tooltip v-if="!canGenerateDescriptionShort">
+                <BrandedButton
+                  type="button"
+                  color="primary"
+                  :disabled="true"
+                >
+                  <div class="flex items-center space-x-2">
+                    <RiSparklingLine
+                      class="size-4"
+                      aria-hidden="true"
+                    />
+                    <span>{{ $t('Suggérer une description courte') }}</span>
+                  </div>
+                </BrandedButton>
+                <template #tooltip>
+                  {{ $t('Remplissez le titre et une description d\'au moins {min} caractères pour utiliser cette fonctionnalité.', { min: DESCRIPTION_MIN_LENGTH }) }}
+                </template>
+              </Tooltip>
+              <BrandedButton
+                v-else
+                type="button"
+                color="primary"
+                :disabled="isGeneratingDescriptionShort"
```

Comment:
You can use the `loading` props instead of the `disabled` one for this, it will disable the button and add a spinner